doc/readthedocs/index.rst

```diff
@@ -38,6 +38,9 @@ This guide to |FOURC| is structured as follows:
 :ref:`Part II - The 4C Community<4Ccommunity>`
    A brief summary of the roles and responsibilities within the |FOURC| community.
 
+:ref:`Installation<Installation>`
+   A summary of all requirements of |FOURC| and detailed steps how to build |FOURC|.
+
 :ref:`Part III — Tutorials<tutorials>`
    A series of beginner-level tutorials showcases the setup procedure for specific application scenarios.
```

Comment:
I did not update the `Part` section because I think this should be rather removed as updating the list means updating the different parts.